ref/config/serverConfiguration.adoc

```diff
@@ -3,13 +3,7 @@
 :page-layout: config
 :page-type: overview
 
-The {projectName} server config is made up of one mandatory file, the
-`server.xml` file, and a set of optional additional files. The `server.xml` is
-the only mandatory config file.  The only requirements for the `server.xml`
-file are that it must be well formed XML and the root element is `server`. When
-processing the `server.xml` file, any elements or attributes that are not
-understood are simply ignored. The default `server.xml` might look like the
-following example:
+The {projectName} server configuration, or server config, is made up of one mandatory file, the `server.xml` file, and a set of optional files. The `server.xml` file must be well-formed XML and the root element must be `server`. When the `server.xml` file is processed, any elements or attributes that are not understood are ignored. The default `server.xml` file might look like the following example:
```

Comment:
I think we should stick with server config. I think config is a widely understood abbreviation for configuration so I don't think we need to clarify.

Reply:
Updated